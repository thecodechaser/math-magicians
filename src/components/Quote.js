import React from "react";

const Quote =()=> {
    return (
        <main class="quote-main">
            <p class="quote">Without mathematics, there’s nothing you can do. Everything around you is mathematics. 
            Everything around you is numbers.
<b>— Shakuntala Devi, Indian writer and mental calculator</b></p>
        </main>
    );
}

export default Quote;