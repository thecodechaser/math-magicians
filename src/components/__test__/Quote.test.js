import { render, cleanup } from '@testing-library/react';
import Quote from '../Quote';

afterEach(() => {
  cleanup();
});

describe('Quote tests', () => {
  it('renders correctly ', () => {
    const tree = render(<Quote />);
    expect(tree).toMatchSnapshot();
  });
});
