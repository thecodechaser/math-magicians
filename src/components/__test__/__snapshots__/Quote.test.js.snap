// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote tests renders correctly  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main
        class="quote-main"
      >
        <p
          class="quote"
        >
          Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
          <b>
            — Shakuntala Devi, Indian writer and mental calculator
          </b>
        </p>
      </main>
    </div>
  </body>,
  "container": <div>
    <main
      class="quote-main"
    >
      <p
        class="quote"
      >
        Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
        <b>
          — Shakuntala Devi, Indian writer and mental calculator
        </b>
      </p>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
